andela/activo-api
# #159651923 Implement endpoint to delete an asset

#### What does this PR do?
Implements an endpoint to delete an asset

#### Description of Task to be completed?
As a developer, I should be able to delete a single asset when I make a request to `/assets/<asset_id>`

**Given** I have a valid authentication token
**When** I make a delete request this endpoint `/assets/<asset_id>` endpoint to delete an asset
**Then** the asset should be deleted
#### How should this be manually tested?

- create a local copy `git fetch origin ft-delete-asset-159651923:ft-delete-asset-159651923`
- checkout to the branch `git checkout ft-delete-asset-159651923`
- Start the application with the command: `flask run`
on postman, send a `DELETE` request to `{{base-url}}/api/v1/assets/<asset_id>`
you should receive a response with status 200 and a message: `"asset deleted successfully"`

#### What are the relevant pivotal tracker stories?
[159651923](https://www.pivotaltracker.com/story/show/159651923)

#### Screenshots 
<img width="1128" alt="screen shot 2018-08-18 at 20 11 51" src="https://user-images.githubusercontent.com/19773670/44301660-8841da00-a323-11e8-82e2-26fe6331e096.png">

#### Documentation
[Postman](https://www.getpostman.com/collections/159071fa20be53f8d8e2)
